index.html

```diff
@@ -137,10 +137,13 @@ <h1>The Registration Process</h1>
   <li>
     Implement at least an experimental version of the new DID Method.
   </li>
-  <li>Create a specification describing the new DID Method.</li>
+  <li>Create a specification describing the new DID Method that is publicly 
+    available and compliant with the DID specification at 
```

Comment:
W3C specs typically talk of "conformance criteria"... and "conforming implementations". It's a fairly high bar -- one that you need a test suite in order to prove that you're compliant. This feels like too high of a bar... perhaps "with the intent of fully conforming to the DID specification"... or something like that.

For example, our current registry includes DID Methods that don't even have a spec yet.

Reply:
The other thing that makes this difficult now is that none of the implementations conform to the DID spec /completely/... but there are a few that are trying really hard. We want to include specs that are trying really hard.

IETF has `provisional` entries in a registry for this purpose. So, you can go `provisional` for years before going `permanent`.

Reply:
Good points. That suggests three things to me.

1. We allow provisional methods, registered with the intent to achieve compliance.
2. Methods should be listed either as PROVISIONAL or COMPLIANT.
3. Until we have a working test suite, all methods are PROVISIONAL

What I don't want is method specs that are not compliant, don't look anything close to being compliant, and probably never will be compliant. The registry shouldn't be the place to propose a radical new deviation from the spec, but rather a place to list those methods that either already conform or which are intended to conform in short order.